ic-helpers/src/management/canister.rs

```diff
@@ -143,6 +145,41 @@ impl Canister {
         .map(|r| Self(r.canister_id))
     }
 
+    pub async fn get_ecdsa_pubkey(
+        principal: Option<Principal>,
+    ) -> Result<Pubkey, (RejectionCode, String)> {
+        let request = GetECDSAPublicKeyArgs {
+            canister_id: principal,
+            derivation_path: vec![],
```

Comment:
We will need to use this parameter in the `neuro` project, so let's have it as an argument to this method. It's not a big deal to give empty vector as an argument when it's not needed.